### DISCUSSION TITLE
更新LicenseKey，Malformed input to a URL function

### DISCUSSION THREAD

**Discussion by dsdffgh** [vvbbnn00/WARP-Clash-API#157] [Q&A]
我尝试更新自己的LicenseKey，我确信host:post是填写对的，但是报错了
当时填写格式
` curl -X POST http://<我的ip>:21001/api/account/update_license -H "Content-Type: application/json" -d "{\"license_key\": \"A******5\"}"`

<!doctype html>
<html lang=en>
<title>400 Bad Request</title>
<h1>Bad Request</h1>
<p>The browser (or proxy) sent a request that this server could not understand.</p>
curl: (3) URL rejected: Malformed input to a URL function

**Comment by chptnt** [comment]
请问你解决了吗，我有同样的问题，当我输入自己的LicenseKey后，提示：{"code":403,"message":"Unauthorized"}

**Comment by dsdffgh** [comment]
> 请问你解决了吗，我有同样的问题，当我输入自己的LicenseKey后，提示：{"code":403,"message":"Unauthorized"}

我也没解决

**Reply by a5412398** [reply]
一样，设置了密码的，不管加不加key=   都是一样的 400

**Reply by a5412398** [reply]
curl -v -k -X POST [https://d.******.xyz:82/api/account/update_license?key=ps*****g001](https://d.%2A%2A%2A%2A%2A%2A.xyz:82/api/account/update_license?key=ps*****g001) -H "Content-Type: application/json" -d "{"license_key": "Kp5fb7*********Y9B6r2"}"
